### PR TITLE
Journey to ENG Blog

### PR DESCRIPTION
**Pull request by bethjaw**
First draft - would love any feedback, it's been a while since I've written anything!

### CODE REVIEW

File: _drafts/2021-11-18-journey-to-engineering-blog.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Communication goes a long way
+
+Aside from the immediate platform knowledge, I had now been calling prospects and talking to customers and I had gained a ton of confidence in communication I didn’t even know I was lacking.  This felt key to being a successful engineering team member due to the ever changing nature of not just being at a startup, but being at one which is in growth mode!  There is a lot going on, there are a lot of changes happening and it’s important to be able to speak up, reach out and voice opinions.  It may spark a needed conversation, ignite a new idea or you might just learn something!
```
Comment (regexpressyourself) [inline _drafts/2021-11-18-journey-to-engineering-blog.md:25]: Might be worth mentioning something about how prepared you would have been here without your specific experience. 

I can only speak to my experience, but I can imagine without my time as a cashier I'd have been much less willing to reach out with questions in my first dev job. Could be cool to highlight how your experience helped ease the transition into engineering.

**Review by regexpressyourself — APPROVED** [review]
I think this is great! The excitement in the tone really comes out, and it shows off an awesome piece of our culture.

I thought it could be cool to highlight some specific things in engineering that your non-standard path uniquely helped prepare you for (thinking along the lines of "show don't tell"), but totally up to you.

Really nice work though! Got an audible "aww" out of me at the end😄

File: _drafts/2021-11-18-journey-to-engineering-blog.md (hunk 1 above)
Comment (stockandawe) [reply _drafts/2021-11-18-journey-to-engineering-blog.md:25]: @regexpressyourself , great suggestion. 

@bethjaw , you might be able to talk about how your experience from CS was a great way to get confidence that details matter in communication. E.g when a customer would report an issue, you learned in CS to ask for as much detail and context as possible. As a software engineer now, you can clearly see how when debugging a customer issue, having the details and context really help get to the bottom of the issue more quickly. 

Another area where your confidence in communication might be helping is when working on migrating existing customers from the legacy proxy setup to the new Cloudflare SSL for SaaS architecture that you have been owning. 

https://developers.cloudflare.com/ssl/ssl-for-saas

**Review by stockandawe — APPROVED** [review]
I really enjoyed reading this @bethjaw

File: _drafts/2021-11-18-journey-to-engineering-blog.md (hunk 1 above)
Comment (stockandawe) [reply _drafts/2021-11-18-journey-to-engineering-blog.md:25]: @regexpressyourself , great suggestion.

@bethjaw , you might be able to talk about how your experience from CS was a great way to get confidence that details matter in communication. E.g when a customer would report an issue, you learned in CS to ask for as much detail and context as possible. As a software engineer now, you can clearly see how when debugging a customer issue, having the details and context really help get to the bottom of the issue more quickly.

Another area where your confidence in communication might be helping is when working on migrating existing customers from the legacy proxy setup to the new Cloudflare SSL for SaaS architecture that you have been owning.

https://developers.cloudflare.com/ssl/ssl-for-saas

**Comment by bethjaw** [comment]
I liked the idea of some examples!  wouldn't mind one last look if you have time @regexpressyourself and then I'll get this posted before the holiday!

**Review by stockandawe — APPROVED** [review]
👍 Nice edits @bethjaw. Looks good!

**Pull request merged**